app/angular/src/builders/start-storybook/index.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { Architect } from '@angular-devkit/architect';
+import { TestingArchitectHost } from '@angular-devkit/architect/testing';
+import { schema } from '@angular-devkit/core';
+import * as path from 'path';
+
+const buildStandaloneMock = jest.fn().mockImplementation((_options: unknown) => Promise.resolve());
+
+jest.mock('@storybook/angular/standalone', () => buildStandaloneMock);
+
+describe('Start Storybook Builder', () => {
+  let architect: Architect;
+  let architectHost: TestingArchitectHost;
+
+  beforeEach(async () => {
+    const registry = new schema.CoreSchemaRegistry();
+    registry.addPostTransform(schema.transforms.addUndefinedDefaults);
```

Comment:
What is this for? (15 and 16) I do lots of stuff with schematics but never have I seen this in tests, I'm curious 🙂

Reply:
no idea. but not nessesary i remove them !
I was inspired by https://angular.io/guide/cli-builder#testing-a-builder ^^

Reply:
this add all default props set in schema.json to test it 

I was inspired by https://angular.io/guide/cli-builder#testing-a-builder ^^